pkg/app/piped/analysisprovider/metrics/prometheus/prometheus.go

```diff
@@ -83,20 +83,28 @@ func (p *Provider) Type() string {
 	return ProviderType
 }
 
-func (p *Provider) RunQuery(ctx context.Context, query string, queryRange metrics.QueryRange, evaluator metrics.Evaluator) (bool, string, error) {
+// Evaluate queries the range query endpoint and checks if values in all data points are within the expected range.
+// For the range query endpoint, see: https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries
+func (p *Provider) Evaluate(ctx context.Context, query string, queryRange metrics.QueryRange, evaluator metrics.Evaluator) (bool, string, error) {
 	if err := queryRange.Validate(); err != nil {
 		return false, "", err
 	}
 
 	ctx, cancel := context.WithTimeout(ctx, p.timeout)
 	defer cancel()
 
+	// NOTE: Use 1m as a step but make sure the "step" isn't smaller than the query range.
```

Comment:
The comment and the code are different doings, aren't they?

Reply:
You're right, what we wanting is totally the opposite; PR title as well.